### ISSUE TITLE
How is the ground truth labeled for this dataset?

### ISSUE THREAD

**Issue by Kylesesa** [haqishen/MFNet-pytorch#9]
I wonder which method you used to label the image as your training ground truth. Labeled the Thermal and RGB images separately, and then intergrated them? Or use some other methods?
